vss_layers/README.md

```diff
@@ -0,0 +1,243 @@
+# VSS-Layers
+
+VSS layers is an extension mechanism that has been implicitly included since
+the VSS development started, and recently there have been more discussions to
+formalize the concept.
+
+Even before the concept had a name it was in practice implemented in the
+initial tools because they behaved like layers in the following ways:
+
+1. Overriding definitions was possible with subsequent re-definition of
+   the same signal
+2. It was possible to add custom metadata fields to each signal, in
+   addition to the VSS "core" model metadata ("name", "type", "datatype",
+   "description", and in recent editions "instances", "deprecation" etc.)
+
+There are two main ways to add a metadata relationship in the plain VSS
+(meta)model.  In the VSSo (ontology) environment there may also be additional
+ways.
+
+The most obvious one is to list new metadata below a signal definition.
+
+VSS core model metadata:
+```
+Motor.CoolantTemperature
+  datatype: int16
+  type: sensor
+  unit: celsius
+  description: Motor coolant temperature (if applicable).
+```
+
+Example of layered additional metadata:
+```
+Motor.CoolantTemperature
+  my_unique_concept: true
+```
+
+### A note about signal names and paths
+
+The VSS is defined by a hierarchy of sub-trees each defined in their own file.
+
+An example file (ElectricMotor.vspec) may include:
+
+```
+Motor.CoolantTemperature
+  ...
+  ...
+```
+
+but since this information is in the file ElectricMotor.vspec, and
+that file was included at the location of Vehicle.Powertrain.ElectricMotor,
+the above shortened definition actually spells out the following full-path
+signal:
+
+`Vehicle.Powertrain.ElectricMotor.Motor.CoolantTemperature`
+
+This principle for file-inclusions and namespacing applies to VSS-layers in
+the same way as the core VSS.  For full details, refer to the VSS
+documentation.
+
+Note however, that later examples in this text will use the full path for
+clarity.  E.g.: `Vehicle.Chassis.Wheel.Brake.Fluidlevel`
+
+## Usage of VSS-layers
+
+VSS-layers is a fully generic extension mechanism that is likely to find new
+uses over time, but some of the primary drivers are:
+
+1. The general separation of a "deployment model" that is not included in the core definition.
+1. Defining supplementary layers that deal with, for example, data classification.
+1. Ability for companies to tie VSS into existing software and processes that needs additional concepts or information
+
+Some of these usages may lead to agreed-upon layers used by the whole industry, a.k.a. standard layers.
+Others will be system, product, or company specific.
+
+### Deployment model
+
```

Comment:
I feel this whole section is a lot of text, without saying much. 

I would suggest keeping paragraph two  `A Deployment Model keeps the information necessary to realize the
 described interface in a particular environment...` and probably adding some more clear examples such as "values" for attributes or min/max metadata